### PR TITLE
Add debug info to explore page

### PR DESCRIPTION
**Pull request by shifucun**
Made a bunch of renaming based on object type as we have more data in this page now.

### CODE REVIEW

**Review by pradh — APPROVED** [review]
This is really helpful Bo!

Adding Jennifer for a quick look so she knows what's changing in this code.

File: static/js/apps/explore/app.tsx [hunk 1]
```diff
@@ -239,27 +252,44 @@ export function App(): JSX.Element {
...
+      if (
+        shouldSetQuery &&
+        pageMetadata.mainTopic?.name &&
+        pageMetadata.place.name
+      ) {
+        const q = `${pageMetadata.mainTopic.name} in ${pageMetadata.place.name}`;
         setQuery(q);
       }
     }
+    console.log(fulfillData);
```
Comment (chejennifer) [inline static/js/apps/explore/app.tsx:279]: remove this console log?

**Review by chejennifer — APPROVED** [review]
nice!

File: static/js/apps/explore/app.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/apps/explore/app.tsx:279]: I will remove in next PR!

**Pull request merged**